src/domain/templates/components/Forms/common/mappings.ts

```
@@ -134,220 +151,285 @@ export const toCreateTemplateMutationVariables = (
+            whiteboardContent: calloutTemplateData.callout?.framing.whiteboard?.content,
+          },
+          contributionDefaults: handleContributionDefaults(calloutTemplateData.callout?.contributionDefaults),
+        },
+      };
       // Delete useless fields and leave only the fields relevant to the callout types
-      switch (calloutDraft.callout?.type) {
+      switch ((template as CalloutTemplate).callout?.type) {
         case CalloutType.Post: {
-          delete calloutDraft.callout?.contributionDefaults;
-          delete calloutDraft.callout?.framing.whiteboard;
+          delete updateCalloutVariables.calloutData?.contributionDefaults;
```

Biome

[error] Avoid the delete operator which can impact performance.



(lint/performance/noDelete)